src/components/SummaryPanel/RpsChart.tsx

```diff
@@ -44,25 +44,31 @@ export default class RpsChart extends React.Component<RpsChartTypeProp, {}> {
     let len = dataRps.length;
     let minRps: number = len > 1 ? +dataRps[1] : 0;
     let maxRps: number = len > 1 ? +dataRps[1] : 0;
-    let sumRps: number = 0;
     for (let i = 2; i < len; ++i) {
       let sample: number = +dataRps[i];
       minRps = sample < minRps ? sample : minRps;
       maxRps = sample > maxRps ? sample : maxRps;
-      sumRps += +sample;
     }
-    const avgRps = len < 2 ? 0 : sumRps / (len - 1);
 
-    len = dataErrors.length;
-    let minErr: number = len > 1 ? +dataErrors[1] : 0;
-    let maxErr: number = len > 1 ? +dataErrors[1] : 0;
-    for (let i = 2; i < len; ++i) {
-      let sample: number = +dataErrors[i];
-      minErr = sample < minErr ? sample : minErr;
-      maxErr = sample > maxErr ? sample : maxErr;
+    let pctMinErr: number = 0;
+    let pctMaxErr: number = 0;
+    if (dataErrors.length > 0 && dataErrors.length !== dataRps.length) {
+      // We should be getting the same sized arrays for both; if not, something is wrong.
+      // We won't blow up, but we'll show NaN to indicate a problem.
+      pctMinErr = NaN;
+      pctMaxErr = NaN;
+    } else {
+      len = dataErrors.length;
+      if (len > 1) {
+        pctMinErr = dataRps[0] && !isNaN(dataRps[0]) ? +dataErrors[0] / +dataRps[0] * 100 : 0;
```

Comment:
The datapoints actually start at index 1 so I think you need to change the indexes here from 0 to 1.

Also, I think this test dataRps[0] && !isNaN(dataRps[0])  ? should just be:
  dataRps[1] != 0

There should be an actual number in there (not NaN) but it can't be 0 to avoid divBy0.